### PR TITLE
[STINS-011] Implement buttons

### PR DESCRIPTION
**Pull request by jzha411**
issue:
Implements buttons in frontend

Solution:
Creating the connections between pages using interactive buttons for the Stick Insect project.

Risk:
No risk.

Reviewed by:
Jason, Jason, Lucas

### CODE REVIEW

**Comment by lucas2005gao** [comment]
Hey Jason!! I havn't thoroughly read through the functional changes of this PR yet but can you please go to the Github Actions and have a look at the errors the ESLint and Prettier is giving you:
image
Click on details and it will tell you

**Comment by lucas2005gao** [comment]
You will need to first npm install again to install the dependencies I added, then run `npm run lint:fix` in the frontend folder!
run `npm run lint` again to see if the everything is passing

**Review by lucas2005gao — CHANGES_REQUESTED** [review]
You will need to first npm install again to install the dependencies I added, then run npm run lint:fix in the frontend folder!
run npm run lint again to see if the everything is passing

**Review by jzha411 — COMMENTED** [review]
Reviewed all changes

**Review by lucas2005gao — APPROVED** [review]
Nice!! Great work!

**Review by aidandayvyd — APPROVED** [review]
LGTM :)

**Review by jekb2019 — APPROVED** [review]

File: frontend/src/components/ConsentPage.js [hunk 1]
```diff
@@ -18,10 +18,15 @@ const ConsentPage = (props) => (
...
     </div>
     <div className={styles.consent_page_agreement_btns}>
-      <button type="button">I AGREE</button>
-      <button type="button">NO, TAKE ME AWAY</button>
+      <button onClick={() => pageChange("question")} type="button">
+        {" "}
+        I AGREE
+      </button>
+      <button onClick={() => pageChange("main")} type="button">
+        {" "}
```
Comment (dlee658) [inline frontend/src/components/ConsentPage.js:26]: Just saying, {" "}, this is added by prettier, that automatically format code.
It does not really matter, but you can remove this line of code.

**Review by dlee658 — APPROVED** [review]

**Pull request merged**